### DISCUSSION TITLE
Laravel application on dokku keeps timing out after some time

### DISCUSSION THREAD

**Discussion by nouce** [dokku/dokku#5552] [Q&A]
Hi Everyone,

I have a laravel application running on dokku. It runs fine, but everytime after a few ours / a day the application keeps timing out. When i do a `dokku ps:restart {app}` everyting work fine again.

Can someone point me in the right direction where to look. I don't know why this keeps happening...

I'm sorry i don't have alot more information.

Thanks in advanced.

**Comment by josegonzalez** [comment]
You might want to look into your log output, and maybe setup some sort of error tracking on your app (getsentry is good). Adding something like pingdom to check when your site is down and alert you will minimize the time it takes to recover. Finally, an APM service such as New Relic or Datadog will help give you information on what is happening in your requests.
